ios/FluentUI/Tooltip/Tooltip.swift

```diff
@@ -242,14 +242,15 @@ open class Tooltip: NSObject, TokenizedControlInternal {
     public var tokenSet: TooltipTokenSet = .init()
     var tokenSetSink: AnyCancellable?
     var fluentTheme: FluentTheme {
-        guard let tooltipView = tooltipViewController?.view else {
+        // Use anchor view to get theme since tooltip view will most likely be nil
+        guard let anchorView = anchorView else {
```

Comment:
Last thought: could the var pointer to `anchorView` down at the bottom of the file also be `weak` for safety? Shouldn't result in any additional changes to the class.